tests/frame_test.go

```diff
@@ -21,3 +23,58 @@ func TestFramePress(t *testing.T) {
 
 	require.Equal(t, "AbC", f.InputValue("#text1", nil))
 }
+
+func TestFrameDismissDialogBox(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "alert",
+		},
+		{
+			name: "confirm",
+		},
+		{
+			name: "prompt",
+		},
+		{
+			name: "beforeunload",
+		},
+	}
```

Comment:
Since all we need is a string, to make it simpler, we can use `[]string` here instead of a `struct`. We can add a struct back if needed, but, currently it's not needed.

```suggestion
	tests := []string{"alert", "confirm", "prompt", "beforeunload"}
```

Reply:
I prefer keeping it as structs. If someone takes a quick look at it, it's clear that this is a table test.

Reply:
Table testing is not related whether you're using a struct or not.

Reply:
Table testing is not related to whether you're using a struct or not 😅

Examples: [1](https://github.com/golang/go/blob/656d3f4401e1fdb628e24027cf91cc803c7a0dac/src/cmd/go/internal/work/exec_test.go#L37) and [2](https://github.com/golang/go/blob/50b8f490e1701773254fdbfe0e3fea00ff1f2a68/src/net/mail/message_test.go#L1029).

Reply:
Oh nice! Thanks for the references. I'll change it.

Reply:
Sure thing :)

Reply:
Resolved in: 40b4a87aefa1bdd6783b4d13368ee49e596ef092